docs/middleware/static.md

```diff
@@ -0,0 +1,163 @@
+---
+id: static
+---
+
+# Static
+
+Static middleware for Fiber that serves static files such as **images**, **CSS,** and **JavaScript**.
+
+:::info
+By default, **Static** will serve `index.html` files in response to a request on a directory. You can change it from [Config](#config)`
+:::
+
+## Signatures
+
+```go
+func New(root string, cfg ...Config) fiber.Handler
+```
+
+## Examples
+
+```go
+package main
+
+import (
+  "github.com/gofiber/fiber/v3"
+  "github.com/gofiber/fiber/v3/middleware/static"
+)
+
+func main() {
+  app := fiber.New()
+  
+  app.Get("/*", static.New("./public"))
+  
+  app.Listen(":3000")
+}
+```
+
+<details>
+<summary>Test</summary>
+
+```sh
+curl http://localhost:3000/hello.html
+curl http://localhost:3000/css/style.css
+```
+
+</details>
+
+```go
+package main
+
+import (
+  "github.com/gofiber/fiber/v3"
+  "github.com/gofiber/fiber/v3/middleware/static"
+)
+
+func main() {
+  app := fiber.New()
+  
+  app.Use("/", static.New("./public"))
+  
+  app.Listen(":3000")
+}
+```
+
+<details>
+<summary>Test</summary>
```

Comment:
please change the headlines to meaningful ones so that everyone knows what kind of example to expect